.github/workflows/build-android.yml

```diff
@@ -23,39 +23,54 @@ jobs:
     name: Build for Android
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
-        submodules: 'true'
-      continue-on-error: true
-    - name: Chocolatey
+        depth: 1
+        submodules: 'false'
+    - name: Update submodules
+      run: |
+        git submodule sync
+        git config --global submodule.lib/modules.update none
+        git -c protocol.version=2 submodule update --init --force --depth=1
+    - name: Setup Java
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'adopt'
+        java-version: '11'
+    - name: Remove default github maven configuration
+      # Workaround for: 'Unable to decrypt local Maven settings credentials'

```

Comment:
Is this workaround caused by some bug?

Reply:
Looks like settings.xml included in the image has some format that not readable  by gradle

https://github.com/mark-vieira/gradle-maven-settings-plugin/issues/15#issuecomment-404548907